### ISSUE TITLE
Autocomplete

### ISSUE THREAD

**Issue by FunnyDevs** [tobykurien/Xtendroid#136]
Hi! I 'm trying xtend and xtendroid samples but i am not able to use autocomplete on overriding fuctions  inside class. When i click with mouse on the suggestion the IDE deletes the First lettera i errore instead of generate the fuctions. IS IT normal? Thanks

**Comment by tobykurien** [comment]
Overridng autocomplete should work, just remember that Xtend uses the `override` keyword instead of `def` when overriding. I use it like this:

```xtend
@AndroidActivity class MyActivity {
   onC<Ctrl+Space>
}```

At the point of pressing Ctrl+Space you should get autocomplete for overriding `OnCreate`, etc.

**Comment by FunnyDevs** [comment]
yes, i get the autocomplete but when i press enter, the "onC" disappers and the IDE doesn't create the functions and deletes all.

I have recorded a video to show you the problem

https://streamable.com/g0yg8

**Comment by tobykurien** [comment]
This looks like a bug in the Xtend plugin. Does it do that if you press Enter instead of clicking with the mouse? You can discuss this bug on the Xtend group here: https://groups.google.com/group/xtend-lang

Also check if you'r'e using the latest stable release here: http://xtend-lang.org

**Comment by FunnyDevs** [comment]
Even if i press enter. I would like to use xtend and xtendroid but with this problem is impossibile. On Gitter they said me that xtend idea plugin is abandoned.

**Comment by tobykurien** [comment]
No it's not abandoned. It's available for both Eclipse and IntelliJ. See the above two links for more. Xtend has reached a high level of maturity which is why there aren't lots of releases. I can say I've been using Xtend since 2013 and haven't encountered your issue, so it may be some installation or configuration issue. Try asking on the Xtend Google group linked above.

**Comment by FunnyDevs** [comment]
no, they said the xtend intellij idea plugin was abandoned ,last release almost 2 years ago. I have downloaded even your xtendroid examples and i have same problem (i use android studio 2.3.2)

**Comment by Buggaboo** [comment]
@FunnyDevs  [this](https://github.com/xtext/xtend-gradle-plugin) has been abandoned.
[Xtext-idea-gradle-plugin](https://github.com/xtext/xtext-gradle-plugin) is alive and kicking.

**Comment by FunnyDevs** [comment]
I found the problem but there is no solution online

' RuntimeException: Error during auto build.: java.io.IOException: Invalid file name: ':' '

It's a common problem with xtend on windows
